src/chart-addons/stacked-legend.js

```diff
@@ -115,27 +132,80 @@ async function getTopValues(layer, layerName, size) {
     return newCf
       .dimension(dimension)
       .group()
-      .topAsync(size, OFFSET)
+      .topAsync(
```

Comment:
Needed to add additional `ORDER BY` clause to make query deterministic and ensure the same data doesn't get returned across multiple queries due to tiebrakers in the `val` column, so need to pass in extra default params